libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -51,11 +51,11 @@
             [id]="_addOnNonButtonId"
             [compact]="compact"
             [placement]="placement"
-            [attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : glyph ? glyph : addOnText"
-            [attr.title]="glyph ? glyph : addOnText"
+            [attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : (glyph ? glyph : addOnText)"
+            [attr.title]="iconTitle? iconTitle: (glyph ? glyph : addOnText)"
```

Comment:
Could be simplified `iconTitle || glyph || addOnText`.

Reply:
Done